src/VizzyUtilsContainer.tsx

```diff
@@ -36,18 +47,27 @@ const vis: SteppedFunnelChart = {
     Object.assign(previousOptions, this.options)
 
     // add any dynamic options
-    this.options.left_axis_label = config.label_left_axis && Vizzy.dependString("Axes", "Left Axis Label", "", "label_left_axis", vis)
+    this.options.strictDims = Vizzy.dependToggle("Plot", `Expect exactly ${config.numDims} dimensions`, false, "numDims", vis)
+    this.options.strictMeas = Vizzy.dependToggle("Plot", `Expect exactly ${config.numMeas} measures`, false, "numMeas", vis)
 
     // register new options if options has changed since last render
     if (JSON.stringify(previousOptions) !== JSON.stringify(this.options)) {
       this.trigger && this.trigger('registerOptions', this.options)
     }
 
+    // use Vizzy to create a slim, data-ful queryResponse
+    const vizData = Vizzy.prepare(data, config, queryResponse, vis, {
+      numDims: config.numDims,
+      strictDims: config.strictDims,
+      numMeas: config.numMeas,
+      strictMeas: config.strictMeas,
+    })
+
     // render chart
     this.chart = ReactDOM.render(
       <ComponentsProvider>
         <JsonViewer 
-          data={data}
+          data={eval(config.view)}
```

Comment:
I haven't looked at this component, but does this allow you to view just the string option from options.view? or is it the whole vis config?
Mainly curious what the use of eval here is, since it's usually a bad idea.